src/components/RadioButtonLarge/RadioButtonLarge.vue

```diff
@@ -32,9 +32,9 @@
     <label
       :for="id"
       :class="[
-        'relative flex type-base-500 -top-[24px] left-[46px] cursor-pointer h-full pr-16',
+        'relative flex type-base-500 top-[1px] left-[46px] cursor-pointer h-full pr-16',
         {'text-gray-400 !cursor-not-allowed': disabled},
-        {'!-top-[22px]' : helperText}
+        {'' : helperText}
```

Comment:
nit, remove empty class line?

Reply:
yeah, missed that - thanks

Reply:
That's why you should use git add -p lol